### PR TITLE
Add find_hits_in_batches

### PR DESCRIPTION
**Pull request by malept**
Adds a convenience method that yields `SearchHits` when scrolling.

### CODE REVIEW

**Comment by AlexMooney** [comment]
Looks good.  Add a test in https://github.com/data-axle/elastic_record/blob/master/test/elastic_record/relation/batches_test.rb ?

**Comment by malept** [comment]
I thought about that, but it's basically tested via `find_in_batches` and `find_ids_in_batches`.

**Comment by AlexMooney** [comment]
But `find_hits_in_batches` is a new public method that content system is using.  I'm sure it works because of those tests, but if someone deletes it and the tests still pass, we could be in a bad way.

File: test/elastic_record/relation/batches_test.rb [hunk 1]
```diff
@@ -16,6 +16,15 @@ def setup
   #   assert_equal [@red_widget, @blue_widget, @green_widget].to_set, results.to_set
   # end
 
+  def test_find_hits_in_batches
+    results = []
+    Widget.elastic_relation.find_hits_in_batches do |hits|
+      results << hits
+    end
+    expected_ids = [[@red_widget, @blue_widget, @green_widget].to_set]
```
Comment (kunruh9) [inline test/elastic_record/relation/batches_test.rb]: These are `SearchHit` objects, right? Not ids.
```suggestion
    expected_hits = [[@red_widget, @blue_widget, @green_widget].to_set]
```

**Pull request merged**